query-engine/core/src/schema/enum_type.rs

```diff
@@ -1,12 +1,13 @@
-use prisma_models::{InternalEnum, OrderBy, ScalarFieldRef};
+use prisma_models::{InternalEnum, ScalarFieldRef};
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub enum EnumType {
-    /// Enum from the internal data model.
-    Internal(InternalEnum),
+    /// Generic, prisma-application specific string enum.
+    /// Semantics are defined by the component interpreting the contents.
+    String(StringEnumType),
 
-    /// Enum defined for order by on fields.
-    OrderBy(OrderByEnumType),
```

Comment:
I'm glad this one is gone now. I was always confused when stumbling upon this one :-D